db/sql/accounts/column_delegates.sql

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
In general, prefer `git mv` so we can see can track which files have been moved where.

Reply:
I moved files visually in Webstorm IDE, presuming it knows what it is doing underneath :)

Reply:
Actually I may have misunderstood how `git mv` works: https://git.wiki.kernel.org/index.php/GitFaq#Why_does_Git_not_.22track.22_renames.3F

Reply:
:smile: I was expecting Webstorm to be doing nothing short of optimal git commands for that :)